cli/remotebazel/remotebazel.go

```diff
@@ -49,14 +51,17 @@ const (
 	escapeSeq                  = "\u001B["
 	gitConfigSection           = "buildbuddy"
 	gitConfigRemoteBazelRemote = "remote-bazel-remote-name"
-	//TODO(Maggie): Make this configurable
-	defaultRemoteExecutionURL = "remote.buildbuddy.io"
+	defaultRemoteExecutionURL  = "remote.buildbuddy.io"
 )
 
 var (
-	execOs            = flag.String("os", "linux", "If set, requests execution on a specific OS.")
-	execArch          = flag.String("arch", "amd64", "If set, requests execution on a specific CPU architecture.")
-	containerImage    = flag.String("container_image", "", "If set, requests execution on a specific runner image. Otherwise uses the default hosted runner version. A `docker://` prefix is required.")
+	remoteFlagset = flag.NewFlagSet("remote", flag.ContinueOnError)
+
+	execOs             = remoteFlagset.String("os", "linux", "If set, requests execution on a specific OS.")
+	execArch           = remoteFlagset.String("arch", "amd64", "If set, requests execution on a specific CPU architecture.")
+	containerImage     = remoteFlagset.String("container_image", "", "If set, requests execution on a specific runner image. Otherwise uses the default hosted runner version. A `docker://` prefix is required.")
+	remoteRunnerTarget = remoteFlagset.String("remote_runner_target", defaultRemoteExecutionURL, "The target the remote runner should run on. Defaults to the prod app server.")
```

Comment:
a couple of nits:
- No need to mention the default value explicitly, since those are automatically displayed in the help output
- Drop `_target` suffix to match Bazel's convention: `remote_executor`, `remote_cache` etc. are also gRPC targets but don't have a `_target` suffix
- Maybe call out that this has to be a BB target because the remote runner service isn't a standardized API.

```suggestion
	remoteRunnerTarget = remoteFlagset.String("remote_runner", defaultRemoteExecutionURL, "The gRPC target providing the BuildBuddy remote runner service.")
```